backend_new/test/integration/user.e2e-spec.ts

```diff
@@ -27,7 +38,8 @@ describe('User Controller Tests', () => {
     await app.close();
   });
 
-  it('should get no users from list() when no params and no data', async () => {
+  // without clearing the db between tests or test suites this is flakes because of other e2e tests
+  it.skip('should get no users from list() when no params and no data', async () => {
```

Comment:
Same as above

Reply:
yeah basically same comment as above, we have some ideas but need to try em out before a discussion + implementation can happen